styles.css

```diff
@@ -0,0 +1,18 @@
+.color-sold-tv {
```

Comment:
Helemaal goed! Het is sowieso 'netter' om classes in CSS te gebruken - een ID moet je eigenlijk alleen gebruiken als een element echt extreem uniek is en het je niet lukt om dat element te targeten.